src/clients/kdump_proposal.rb

```diff
@@ -41,7 +41,24 @@ def main
 
       if @func == "MakeProposal"
         Kdump.Propose
-        @ret = { "raw_proposal" => Kdump.Summary }
+
+        @ret = {
+          "raw_proposal" => Kdump.Summary,
+          # FATE#317488 When expectation at the end of proposal does not match
+          # the value, this proposal will be called again
+          "trigger"      => {
+            "expect"     => "
+              Yast.import \"Kdump\"
+              Yast::Kdump.free_space_for_dump
+            ",
```

Comment:
To be fair, FunRef documentation is quite terse: [FunRef](http://www.rubydoc.info/github/yast/yast-ruby-bindings/Yast/FunRef), [fun_ref](http://www.rubydoc.info/github/yast/yast-ruby-bindings/Yast#fun_ref-instance_method). I'll improve it.